package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "gatsby-plugin-meilisearch",
+  "version": "0.1.0",
+  "description": "A plugin to index your Gatsby content to MeiliSearch based on graphQL queries",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint --ext .js ."
```

```suggestion
    "lint": "eslint --ext .js .",
    "lint:fix": "eslint --ext .js . --fix"
    
```

Add fixing script (could you confirm it works?)

---

It does with the proper plugins. Just added it to an other commit 😉